### PR TITLE
Sidetable

### PR DESCRIPTION
**Pull request by chris1610**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/sidetable/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 44695ab38e7e166664bf79cf5af23f4360f4d9a5e17114b5ebc4a6908f8be3d9
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pandas >=1.0
```
Comment (scopatz) [inline recipes/sidetable/meta.yaml:18]: Do you need pandas to build this package?

File: recipes/sidetable/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 44695ab38e7e166664bf79cf5af23f4360f4d9a5e17114b5ebc4a6908f8be3d9
+
+build:
+  number: 0
```
Comment (scopatz) [inline recipes/sidetable/meta.yaml:13]: This looks like it can be `noarch: python`

File: recipes/sidetable/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    - seaborn
+
+about:
+  home: "https://github.com/chris1610/sidetable"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: "sidetable builds simple but useful summary tables of your data"
+  doc_url: 
+  dev_url:
```
Comment (scopatz) [inline recipes/sidetable/meta.yaml:39]: Please remove empty entries

File: recipes/sidetable/meta.yaml (hunk 1 above)
Comment (chris1610) [reply recipes/sidetable/meta.yaml:18]: No. To use it yes but not to build it. I will fix and re-submit.

File: recipes/sidetable/meta.yaml (hunk 2 above)
Comment (chris1610) [reply recipes/sidetable/meta.yaml:14]: Got it. Will do.

File: recipes/sidetable/meta.yaml (hunk 3 above)
Comment (chris1610) [reply recipes/sidetable/meta.yaml]: Done. Thx

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**